source/templates.tex

```diff
@@ -8678,8 +8678,8 @@
 struct A { @\commentellip@ };
 struct B : A { @\commentellip@ };
 void g(A a, B b) {
-  f(a,b);           // error: \tcode{T} could be \tcode{A} or \tcode{B}
-  f(b,a);           // error: \tcode{T} could be \tcode{A} or \tcode{B}
+  f(a,b);           // error: \tcode{T} can be \tcode{A} or \tcode{B}
+  f(b,a);           // error: \tcode{T} can be \tcode{A} or \tcode{B}
```

Comment:
I don't think this is right; it can't be either because this is ill-formed. How about:

"error: `T` deduced as both `A` and `B`"

Reply:
fixed